src/cloud.c

```diff
@@ -515,6 +516,17 @@ int cloud_set_read_handler(cloud_cb_t read_handler, void *user_data)
 
 	amqp_bytes_free(queue_fog);
 
+	queue_reply = mq_declare_new_queue(MQ_QUEUE_REPLY);
+	if (queue_reply.bytes == NULL) {
+		l_error("Error on declare a new queue.\n");
```

Comment:
Please, remove the '.' (and the '\n') character.

Reply:
Done